aspnetcore/signalr/javascript-client.md

```diff
@@ -60,18 +65,19 @@ The client library is available on the following CDNs:
 
 The following code creates and starts a connection. The hub's name is case insensitive:
 
-[!code-javascript[](javascript-client/samples/3.x/SignalRChat/wwwroot/chat.js?range=3-6,29-45)]
+[!code-javascript[](javascript-client/samples/6.x/SignalRChat/wwwroot/chat.js?range=3-6,29-45)]
 
-### Cross-origin connections
+### Cross-origin connections (CORS)
```

Comment:
Can we cross link somewhere in this section to more specifics about SignalR and CORS:
https://docs.microsoft.com/aspnet/core/signalr/security?view=aspnetcore-6.0#cross-origin-resource-sharing